app-creator/data/data.go

```diff
@@ -14,6 +14,7 @@ import (
 */
 type Template struct {
 	Id string `json:"id"` 
+	Version float32 `json:"version"` 
```

Comment:
Use a string instead to handle the double dots in the version.
Side note: how do we keep the templates stored in the datastore in sync with the ones served as a backup? We can chat about this offline.